go.mod

```diff
@@ -2,51 +2,52 @@ module github.com/pulumi/pulumi-yaml
 
 go 1.19
 
+replace github.com/hashicorp/vault/api v1.8.2 => github.com/hashicorp/vault/api v1.9.0
+
+replace google.golang.org/protobuf v1.33.0 => google.golang.org/protobuf v1.31.0
```

Comment:
This is the most awkward think imho about this whole PR.  The protobuf 1.33.0 library causes a circular dependency, so we need to downgrade here.